### PR TITLE
DOC-2239 delete RERC

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: DOC-2239
Staging: https://docs.redis.com/staging/DOC-2239/kubernetes/delete_custom_resources/

### CODE REVIEW

File: content/kubernetes/delete_custom_resources.md [hunk 1]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
   kubectl get reaadb -o=jsonpath='{range .items[*]}{.metadata.name}'
   ```
 
+## Delete a remote cluster (RERC)
+
+{{<note>}}The Redis Enterprise Active-Active database (REAADB) custom resource is currently in public preview. View [Preview REAADB]({{<relref "/kubernetes/active-active/preview/">}}) for more details.{{</note>}}
+
+1. Verify the RERC you want to delete isn't listed as a particpating cluster in any REAADB resources.
+
+  If an RERC is still listed as a participating cluster in any database, the deletion will fail.
```
Comment (rrelledge) [inline content/kubernetes/delete_custom_resources.md:90]: The numbered list isn't rendering correctly. I think adding additional spaces to increase the indentation will fix it. This is also an issue for [Delete an Active-Active database](https://docs.redis.com/staging/DOC-2239/kubernetes/delete_custom_resources/#delete-an-active-active-database-reaadb).
```suggestion
    If an RERC is still listed as a participating cluster in any database, the deletion will fail.
```

File: content/kubernetes/delete_custom_resources.md [hunk 2]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
...
+
+1. On one of the existing participating clusters, delete the RERC (substituting `<rerc-name>` with your database name).
+
+  ```sh
```
Comment (rrelledge) [inline content/kubernetes/delete_custom_resources.md:94]: Related to the numbered list issue:
```suggestion
    ```sh
```

File: content/kubernetes/delete_custom_resources.md [hunk 3]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
...
+  kubectl delete rerc <rerc-name>
+  ```
```
Comment (rrelledge) [inline content/kubernetes/delete_custom_resources.md:96]: Related to the numbered list issue:
```suggestion
    ```
```

File: content/kubernetes/delete_custom_resources.md [hunk 4]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
...
+
+1. Verify the RERC no longer exists.
+
+  ```sh
```
Comment (rrelledge) [inline content/kubernetes/delete_custom_resources.md:100]: Related to the numbered list issue:
```suggestion
    ```sh
```

File: content/kubernetes/delete_custom_resources.md [hunk 5]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
...
+  kubectl get rerc -o=jsonpath='{range .items[*]}{.metadata.name}'
+  ```
```
Comment (rrelledge) [inline content/kubernetes/delete_custom_resources.md:102]:
```suggestion
    ```
```

**Review by rrelledge — COMMENTED** [review]
Looks good to me. I left some small suggestions, mostly related to fixing the rendering of numbered lists.

**Comment by kaitlynmichael** [comment]
@yuvallevy2 @alon-zada the section on deleting the RERC still needs review: https://docs.redis.com/staging/DOC-2239/kubernetes/delete_custom_resources/#delete-a-remote-cluster-rerc

**Review by cmilesb — APPROVED** [review]

File: content/kubernetes/delete_custom_resources.md [hunk 6]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
   kubectl get reaadb -o=jsonpath='{range .items[*]}{.metadata.name}'
   ```
 
+## Delete a remote cluster (RERC)
+
+{{<note>}}The Redis Enterprise Active-Active database (REAADB) custom resource is currently in public preview. View [Preview REAADB]({{<relref "/kubernetes/active-active/preview/">}}) for more details.{{</note>}}
+
+1. Verify the RERC you want to delete isn't listed as a particpating cluster in any REAADB resources.
+
+    If an RERC is still listed as a participating cluster in any database, the deletion will fail.
```
Comment (alon-zada) [inline content/kubernetes/delete_custom_resources.md:90]: maybe use this wording:
"""
the deletion will fail
"""
as:
the deletion will be blocked

**Review by alon-zada — DISMISSED** [review]
Great!

File: content/kubernetes/delete_custom_resources.md [hunk 7]
```diff
@@ -78,6 +81,26 @@ You will also need to remove [the `namespaceSelector` section from the validatin
   kubectl get reaadb -o=jsonpath='{range .items[*]}{.metadata.name}'
   ```
 
+## Delete a remote cluster (RERC)
+
+{{<note>}}The Redis Enterprise Active-Active database (REAADB) custom resource is currently in public preview. View [Preview REAADB]({{<relref "/kubernetes/active-active/preview/">}}) for more details.{{</note>}}
+
+1. Verify the RERC you want to delete isn't listed as a particpating cluster in any REAADB resources.
```
Comment (kaitlynmichael) [inline content/kubernetes/delete_custom_resources.md:88]:
```suggestion
1. Verify the RERC you want to delete isn't listed as a participating cluster in any REAADB resources.
```

File: content/kubernetes/delete_custom_resources.md (hunk 6 above)
Comment (kaitlynmichael) [inline content/kubernetes/delete_custom_resources.md:90]:
```suggestion
    If an RERC is still listed as a participating cluster in any database, the deletion will be blocked.
```

**Pull request merged**